sechub-solution/docker/run.sh

```diff
@@ -1,119 +1,175 @@
-        storage_options="$storage_options -Dsechub.storage.s3.accesskey=$S3_ACCESSKEY"
-        storage_options="$storage_options -Dsechub.storage.s3.secretkey=$S3_SECRETKEY"
-
-        echo "Object storage:"
-        echo " * Endpoint: $S3_ENDPOINT"
-        echo " * Bucketname: $S3_BUCKETNAME"
-        echo " * Accesskey: $S3_ACCESSKEY"
-    fi
-
-    echo "Upload source code maximum bytes: $SECHUB_MAX_FILE_UPLOAD_SIZE"
-    echo "Upload binaries maximum bytes: $SECHUB_UPLOAD_BINARIES_MAXIMUM_BYTES"
-    echo "Activated profiles: $profiles"
-
-    echo "Java version:"
-    java --version
-
-    java $JAVA_DEBUG_OPTIONS $database_options \
-        $storage_options \
-        -Dfile.encoding=UTF-8 \
-        -Dspring.profiles.active="$profiles" \
-        -Dsechub.targettype.detection.intranet.hostname.endswith=intranet.example.org \
-        -Dsechub.config.trigger.nextjob.initialdelay=0 \
-        -Dsechub.initialadmin.userid="$ADMIN_USERID" \
-        -Dsechub.initialadmin.email=sechubadm@example.org \
-        -Dsechub.initialadmin.apitoken="$ADMIN_APITOKEN" \
-        -Dspring.servlet.multipart.max-file-size="$SECHUB_MAX_FILE_UPLOAD_SIZE" \
-        -Dspring.servlet.multipart.max-request-size="$SECHUB_MAX_FILE_UPLOAD_SIZE" \
-        -Dsechub.upload.binaries.maximum.bytes="$SECHUB_UPLOAD_BINARIES_MAXIMUM_BYTES" \
-        -Dsechub.adapter.netsparker.userid=abc \
-        -Dsechub.adapter.netsparker.apitoken=xyz \
-        -Dsechub.adapter.netsparker.baseurl=https://example.org \
-        -Dsechub.adapter.netsparker.defaultpolicyid=example \
-        -Dsechub.adapter.netsparker.licenseid=example \
-        -Dsechub.adapter.nessus.defaultpolicyid=example \
-        -Dsechub.notification.email.administrators=example@example.org \
-        -Dsechub.notification.email.from=example@example.org \
-        -Dsechub.notification.smtp.hostname=example.org \
-        -Dserver.port=8443 \
-        -Dserver.address=0.0.0.0 \
-        -jar /sechub/sechub-server*.jar
+check_setup() {
+  check_variable "$ADMIN_USERID" "ADMIN_USERID"
+  check_variable "$ADMIN_APITOKEN" "ADMIN_APITOKEN"
+  check_variable "$SECHUB_STORAGE_SHAREDVOLUME_UPLOAD_DIR" "SECHUB_STORAGE_SHAREDVOLUME_UPLOAD_DIR"
 }
 
-check_setup () {
-    check_variable "$ADMIN_USERID" "ADMIN_USERID"
-    check_variable "$ADMIN_APITOKEN" "ADMIN_APITOKEN"
-    check_variable "$SECHUB_STORAGE_SHAREDVOLUME_UPLOAD_DIR" "SECHUB_STORAGE_SHAREDVOLUME_UPLOAD_DIR"
+wait_loop() {
+  while true
+  do
+    echo "wait_loop(): Sleeping for $SLEEP_TIME_IN_WAIT_LOOP."
+    sleep $SLEEP_TIME_IN_WAIT_LOOP
+  done
 }
 
-check_variable () {
-    value="$1"
-    name="$2"
+init_scheduler_settings() {
+  if [ -z "$SECHUB_CONFIG_TRIGGER_NEXTJOB_DELAY" ] ; then
+    export SECHUB_CONFIG_TRIGGER_NEXTJOB_DELAY="10000"
+    echo "---> SECHUB_CONFIG_TRIGGER_NEXTJOB_DELAY is undefined. Falling back to $SECHUB_CONFIG_TRIGGER_NEXTJOB_DELAY"
+  fi
+
+  # Randomize initial delay used for trigger next job
```

Comment:
I neither understand the comment nor really the code. What is the main point of all of this?

Reply:
Please see documentation: https://mercedes-benz.github.io/sechub/latest/sechub-operations.html#section-general-configuration 
Look into Table 14

Reply:
The point is not the trigger next job mechanism. My problem is, I do not understand why you are calculating with 303. What is so special about 303. Is this just an arbitrary value? What do you want to achieve with this?

In addition, the `shuf` command needs a bit of explanation `shuf -i 1-10 -n 1` creates a random value between 1 and 10.

Reply:
The code stems from @de-jcup ; So I don't know the thoughts about 303.
As I understood it, the code creates a random initial wait interval, so when containers start at the same time, this tries to avoid race conditions regarding e.g. db schema upgrades.

I'll check this and try to simplify if possible.

Reply:
done.